### PR TITLE
Added New Account Console Tests to CI workflow.

### LINKED ISSUE #16038: Add account2 tests to Keycloak CI
Account2 is not tested in the base testsuite, but rather in a separate module (other/base-ui). There's also some login tests in this module.

We should enable account2 testing in GitHub Actions. Question is if we should keep these as a separate module or move them to the base tests.

Also, find it strange that we have two login forms tests here, when the majority of login forms testing is in the base tests.

### PR DESCRIPTION
**Pull request by miquelsi**
Closes #16038 and #16515

### CODE REVIEW

File: testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/InternationalizationTest.java [hunk 1]
```diff
@@ -19,6 +19,7 @@
```
Comment (miquelsi) [inline testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/InternationalizationTest.java:19]: Tests ignored as suggested by @ssilvert on this [comment](https://github.com/keycloak/keycloak/issues/16515#issuecomment-1397134888).

**Comment by stianst** [comment]
> testPersistenceMultipleNodesClientSessionAtSameNode

Reported https://github.com/keycloak/keycloak/issues/16571

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -438,6 +438,42 @@ jobs:
         with:
           job-name: FIPS IT
 
+  account-console-tests:
+    name: Account Console Tests
```
Comment (stianst) [inline .github/workflows/ci.yml:442]:
```suggestion
    name: Account Console IT
```

**Review by stianst — COMMENTED** [review]
We should also add account-console-tests to needs for `check-set-status`, otherwise it will be ignored from branch protection checks.

**Review by stianst — APPROVED** [review]

**Pull request merged**